src/types/payload.ts

```diff
@@ -0,0 +1,212 @@
+// https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads
+
+import { Static, Type } from "@sinclair/typebox";
+import { LabelSchema } from "./label";
+import { TURL } from "./shared";
+
+export enum Action {
+  REQUESTED = "requested",
+  REVIEW_REQUESTED = "review_requested",
+  REVIEW_REQUEST_REMOVED = "review_request_removed",
+  COMPLETED = "completed",
+  REREQUESTED = "rerequested",
+
+  ASSIGNED = "assigned",
+  CLOSED = "closed",
+  CREATED = "created",
+  DELETED = "deleted",
+  MILESTONED = "milestoned",
+  DEMILESTONED = "demilestoned",
+  EDITED = "edited",
+  LABELED = "labeled",
+  LOCKED = "locked",
+  OPENED = "opened",
+  PINNED = "pinned",
+  REOPENED = "reopened",
+  TRANSFERRED = "transferred",
+  UNASSIGNED = "unassigned",
+  UNLABELED = "unlabeled",
+  UNLOCKED = "unlocked",
+  UNPINNED = "unpinned",
+}
```

Comment:
Curious what the purpose of the enum is. Why not use the strings directly? Aren't they simplier for typechecking via something like: 

```typescript
export type Action = "requested" | "review_requested" | "review_request_removed" ...
```

Reply:
we can define in both ways but if its type should be used in other schemas, `Enum` was sometimes better with sinclair Type.Enum like 
```Typescript
received:events_url: TURL,
type: Type.Enum(UserType)
```

Reply:
I don't have a strong opinion either way. I ask out of curiosity.